dashboard/api.py

```diff
@@ -509,11 +510,25 @@ def is_exam_schedulable(user, course):
         user=user,
         status=ExamAuthorization.STATUS_SUCCESS,
         exam_run__in=schedulable_exam_runs,
-        exam_coupon__isnull=False
     ).exclude(
         operation=ExamAuthorization.OPERATION_DELETE).exists()
 
 
+def get_exam_schedulable_dates(course):
+    """
+    Get a formatted string of dates during which the exam is schedulable
+    """
+    now = now_in_utc()
+    schedulable_exam_run = ExamRun.objects.filter(
+        course=course, date_last_eligible__gte=now.date()
+    ).first()
+    if schedulable_exam_run is not None:
+        return '{} - {}'.format(
+            schedulable_exam_run.date_first_schedulable.strftime("%B %d"),
+            schedulable_exam_run.date_last_schedulable.strftime("%B %d, %Y"))
```

Comment:
In the screenshot, these date strings had a leading zero on the day of the month. Can we omit that using a different strftime setting?